spopt/__init__.py

```diff
@@ -3,4 +3,4 @@
 
 
 # import modules/functions
-from .region import MaxPHeuristic, RegionKMeansHeuristic, w_to_g
+from .region import MaxPHeuristic, RegionKMeansHeuristic, SKATER, w_to_g
```

Comment:
Should we also expose `Spanning_Forest` here and in `region/__init__.py`?

Reply:
can we commit to pep8 from the get-go in this package? I'm having a tough time trying to convert some of the old packages (so `SpanningForest` if you want it) ?

Reply:
also probably `Skater`...

Reply:
> can we commit to pep8 from the get-go in this package? I'm having a tough time trying to convert some of the old packages (so SpanningForest if you want it) ?

So do you mean:

```python
from region import MaxPHeuristic, RegionKMeansHeuristic, SKATER, w_to_g
```

instead of

```python
from .region import MaxPHeuristic, RegionKMeansHeuristic, SKATER, w_to_g
```
?

Reply:
no i just mean all classes should be camel cased

Reply:
> no i just mean all classes should be camel cased

I really over-thought that one...